commands/list.js

```diff
@@ -9,13 +9,22 @@ var LIST_TYPES = {
 		
 		console.log('Channels:', responseJSON);
 	},
+
 	segments: function _segmentActionHandler (error, response, body) {
 		if (error) return console.log('Error: ', error);
 
 		var responseJSON = JSON.parse(body);
 		
 		console.log('Segments:', responseJSON);
 	},
+
+	named_users: function _namedUsersActionHandler (error, response, body) {
```

Comment:
You should remove all `_` from action handler functions, because they are included into the the object;